index.js

```diff
@@ -24,7 +24,7 @@ module.exports = (robot) => {
     }
   });
 
-  robot.on('schedule.repository', async context => {
+  robot.on('push', async context => {

```

Comment:
First of all, this is completely wrong and will render the merge conflict check unusable. Did you go through the [postmortem of the CLA check](https://docs.google.com/document/d/11DJBTlFqjsWU4oSThWNJKwLsaULcVGMOZVC3psZkHLM/edit#) which we released pretty recently? This is exactly what went wrong then.
Secondly, we have also added a checklist to ensure that all changes are manually tested. It is the responsibility of the PR author to fill it. _No untested PRs will be accepted, even if it involves a minor change_.